src/vms/avm/api.ts

```diff
@@ -33,4 +34,8 @@ export class AVMApi extends AvaxApi {
       getAllBalancesParams,
     );
   }
+
+  getTxFee(): Promise<GetTxFeeResponse> {
+    return this.callRpc<GetTxFeeResponse>('getTxFee');
+  }
```

Comment:
I wonder if it's worth converting the response values to BigInt here like the original info API was doing so you don't have to do that in other places.